src/Components/DashBoard/Repository/RepoComponents/RepoDetails/RepoDetailBackdrop/RemoteConfigComponent/RemoteManagementComponent.js

```diff
@@ -48,124 +66,143 @@ export default function AddRemoteRepoComponent({ repoId }) {
-        <div className="w-11/12 mx-auto">
-          {remoteDetails.length > 0 ? (
-            <>
-              <div className="flex items-center w-full">
-                <div className="font-sans w-1/4 xl:text-2xl lg:text-2xl md:text-xl text-lg mx-auto text-center font-semibold text-gray-600">
-                  Remote name
-                </div>
-                <div className="font-sans xl:text-2xl lg:text-2xl md:text-xl text-lg w-7/12 mx-auto text-center font-semibold text-gray-600">
-                  Remote URL
-                </div>
+      ) : (
+        <>
+          {addRemoteStatus
+            ? statusPillComponent(
+                "border-red-800",
+                "bg-red-200",
+                "text-red-900",
+                "Remote name already exist!"
+              )
+            : null}
+          {fieldMissing
+            ? statusPillComponent(
+                "border-indigo-800",
+                "bg-indigo-200",
+                "text-indigo-900",
+                "One or more required parameters are empty!"
+              )
+            : null}
+          {invalidUrl
+            ? statusPillComponent(
+                "border-yellow-800",
+                "bg-yellow-200",
+                "text-yellow-900",
+                "URL with whitespace is invalid!"
+              )
+            : null}
+          {deleteFailed
+            ? statusPillComponent(
+                "border-red-800",
+                "bg-red-200",
+                "text-red-900",
+                "Failed to delete"
+              )
+            : null}
+          <div className="w-full p-2 pt-6 pb-8">
+            <div className="flex items-center m-6 font-sans text-xl font-semibold text-gray-800 xl:text-3xl lg:text-3xl md:text-2xl sm:text-xl">
+              <FontAwesomeIcon
+                icon={faCodeBranch}
+                className="mx-2 text-xl xl:text-3xl lg:text-3xl md:text-2xl sm:text-xl"
+              ></FontAwesomeIcon>
+              <div className="pb-2 border-b-4 border-blue-400 border-dashed">
+                Remote details
+              </div>
+              {addNewRemote && remoteDetails.length > 0 ? (
                 <div
-                  className="font-sans xl:text-2xl lg:text-2xl md:text-xl text-lg mx-auto text-center font-semibold text-gray-600"
-                  style={{ width: "22%" }}
+                  className="px-3 py-2 mx-6 font-sans text-base text-gray-800 bg-blue-200 rounded cursor-pointer xl:text-lg lg:text-lg md:text-base hover:bg-blue-300 hover:text-gray-900"
+                  onClick={() => {
+                    setAddNewRemote(false);
+                    setRemoteForm(true);
+                  }}
                 >
-                  Actions
+                  Add new remote
                 </div>
-              </div>
-              {remoteForm ? (
+              ) : null}
+            </div>
+            <div className="w-11/12 mx-auto">
+              {remoteDetails.length > 0 ? (
```

Comment:
always check for null before accessing the items from objects `remoteDetails && remoteDetails.length`